seacrowd/sea_datasets/crosssum/crosssum.py

```diff
@@ -0,0 +1,140 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import Dict, List, Tuple
+
+import datasets
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Licenses, Tasks
+
+_CITATION = """
+@inproceedings{bhattacharjee-etal-2023-crosssum,
+  author    = {Bhattacharjee, Abhik and Hasan, Tahmid and Ahmad, Wasi Uddin and Li, Yuan-Fang and Kang, Yong-Bin and Shahriyar, Rifat},
+  title     = {CrossSum: Beyond English-Centric Cross-Lingual Summarization for 1,500+ Language Pairs},
+  booktitle = {Proceedings of the 61st Annual Meeting of the Association for Computational Linguistics},
+  publisher = {Association for Computational Linguistics},
+  year      = {2023},
+  url       = {https://aclanthology.org/2023.acl-long.143},
+  doi       = {10.18653/v1/2023.acl-long.143},
+  pages     = {2541--2564},
+  }
+"""
+
+_LOCAL = False
+_LANGUAGES = ["ind", "mya", "vie"]
+_DATASETNAME = "crosssum"
+_DESCRIPTION = """
+This is a large-scale cross-lingual summarization dataset containing article-summary samples in 1,500+ language pairs,
+including pairs with the Burmese, Indonesian and Vietnamese languages. Articles in the first language are assigned
+summaries in the second language.
+"""
+
+_HOMEPAGE = "https://huggingface.co/datasets/csebuetnlp/CrossSum"
+_LICENSE = Licenses.CC_BY_NC_SA_4_0.value
+_URL = "https://huggingface.co/datasets/csebuetnlp/CrossSum"
+
+
+_SUPPORTED_TASKS = [Tasks.SUMMARIZATION]
```

Comment:
Could you please pull from `main` and change this to `[Tasks.CROSS_LINGUAL_SUMMARIZATION]`?